examples/tensorflow/summarization/run_summarization.py

```diff
@@ -67,6 +67,14 @@
             "Offline mode: run this script without TRANSFORMERS_OFFLINE first to download nltk data files"
         )
     with FileLock(".lock") as lock:
+        try:
+            tokenizers_path = nltk.data.find("tokenizers").path
+        except (LookupError, OSError):
+            tokenizers_path = os.path.join(nltk.data.path[0], "tokenizers")
+            os.makedirs(tokenizers_path, exist_ok=True)
+        punkt_path = os.path.join(tokenizers_path, "punkt")
+        if os.path.exists(punkt_path):
+            os.remove(punkt_path)
```

Comment:
waaaahhhh? we really shouldn't need to do this :/ 

Do we know why this is needed? e.g. examples interacting with each other / updated nltk

Reply:
Absolutely no idea why that happens, but let me try something else before merging this atrocity

Reply:
there we go, out with it - this test is weirdly flaky

Reply:
@amyeroberts removed this, think we can merge and see if this pops again on `main`